model/src/main/java/jetbrains/jetpad/model/transform/Transformers.java

```diff
@@ -825,6 +825,76 @@ protected void doRemove() {
     };
   }
 
+  /**
+   * Select only those with the highest priority. Warning: target collection
+   * is not protected from outside writes.
+   * @param getPriority The greater is number, the higher is priority.
```

Comment:
Some clarification about null in "from" collection and null as a result of getPriority.apply() should be added